### ISSUE TITLE
No module named 'bert_dp' after successful installation

### ISSUE THREAD

**Issue by sokolov0** [deepmipt/DeepPavlov#1005]
I ran
`python -m deeppavlov install ner_rus_bert`
It ends up with
`Successfully built bert-dp`
Then I ran
```
from deeppavlov import configs, build_model

ner_model = build_model(configs.ner.ner_rus_bert, download=True)

ner_model(['Привет. Это текст'])
```
It ends up with
`ModuleNotFoundError: No module named 'bert_dp'`

Relevant detail: I use conda

How do I correct this?

UPD: `conda list` and `pip list` contain bert_dp

**Comment by yurakuratov** [comment]
Hi!

How do you run this code snippet (with `from deeppavlov ...`)?
Do you use the same python as when you ran `python -m deeppavlov install ner_rus_bert`?

**Comment by yoptar** [comment]
You can run `python -c 'import sys; print(sys.executable)'` from a terminal to determine the exact executable used. And in jupyter you can also run
```
import sys
print(sys.executable)
```

**Comment by sokolov0** [comment]
> You can run `python -c 'import sys; print(sys.executable)'` from a terminal to determine the exact executable used. 

It gives me
`C:\Users\ashlyapin\miniconda3\envs\deeppavlov\python.exe`

> And in jupyter you can also run
> 
> ```
> import sys
> print(sys.executable)
> ```
It gives me
`C:\Users\ashlyapin\miniconda3\python.exe`

**Comment by sokolov0** [comment]
I solved the problem. Actually, it was anaconda-related problem.
I ran Anaconda Navigator, installed Notebook in the environment, opened it. After this 

```
import sys
print(sys.executable)
```
gave me `C:\Users\ashlyapin\miniconda3\envs\deeppavlov\python.exe`
And the model works.
Also, if I run 

```
(base) C:\>conda activate deeppavlov
(deeppavlov) C:\>jupyter notebook
```
`sys.executable` also gives me `C:\Users\ashlyapin\miniconda3\envs\deeppavlov\python.exe` and the model works too.
Thank you for help!
